### DISCUSSION TITLE
PrimeVue updates broke my system

### DISCUSSION THREAD

**Discussion by atilaacedo** [primefaces/community#293] [PrimeVue]
PrimeVue updates broke my system, some inputs broke aesthetically on the screens and I would like to know if there is an easier way to solve this

**Comment by cagataycivici** [comment]
A sample reproducer will help, it may be due to CSS layer migration, some other CSS may be breaking the PrimeVue components, best way to track it is open the dev tools and see who overrides the components. Do you have normalize.css, boostrap, tailwind or similar?
